### ISSUE TITLE
False positive for rxjs-no-unsafe-scope on parameter type

### ISSUE THREAD

**Issue by rgant** [cartant/rxjs-tslint-rules#102]
rxjs-tslint-rules is flagging `UserCredential` type the snippet below as an unsafe scope error. I think it shouldn't flag parameter types for this error.

```
const signInPromise = this.afAuth.auth.signInWithEmailAndPassword(email, password);
return from(signInPromise).pipe(
  map((credentials: auth.UserCredential) => credentials.user != undefined),
);
```

`150:30  error  tslint:rxjs-no-unsafe-scope          Unsafe scopes are forbidden`

**Comment by cartant** [comment]
This should be fixed in 4.24.3.
